Tests/SonarScanner.MSBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -267,6 +261,127 @@ public void PreProc_EndToEnd_SuccessCase_With_Organization()
         }
 
+        [DataTestMethod]
+        [DataRow("6.7.0.22152")]
+        [DataRow("7.0.0.22152")]
+        public void PreProc_EndToEnd_ShouldWarn_SonarQubeDeprecatedVersion(string sqVersion)
+        {
+            // Arrange
+            var workingDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var logger = new TestLogger();
+
+            // Configure the server
+            var mockServer = new MockSonarQubeServer();
+
+            var data = mockServer.Data;
+            data.ServerProperties.Add("server.key", "server value 1");
+            data.SonarQubeVersion = new Version(sqVersion);
+
+            data.Languages.Add("cs");
+            data.Languages.Add("vbnet");
+            data.Languages.Add("another_plugin");
+
+            data.AddQualityProfile("qp1", "cs", "organization")
+                .AddProject("key")
+                .AddRule(new SonarRule("csharpsquid", "cs.rule3"));
+
+            data.AddQualityProfile("qp2", "vbnet", "organization")
+                .AddProject("key")
+                .AddRule(new SonarRule("vbnet", "vb.rule3"));
+
+            var mockAnalyzerProvider = new MockRoslynAnalyzerProvider
+            {
+                SettingsToReturn = new AnalyzerSettings
+                {
+                    RuleSetFilePath = "c:\\xxx.ruleset"
+                }
+            };
+
+            var mockTargetsInstaller = new Mock<ITargetsInstaller>();
+            var mockFactory = new MockObjectFactory(mockServer, mockTargetsInstaller.Object, mockAnalyzerProvider);
+
+            TeamBuildSettings settings;
+            using (PreprocessTestUtils.CreateValidNonTeamBuildScope())
+            using (new WorkingDirectoryScope(workingDir))
+            {
+                settings = TeamBuildSettings.GetSettingsFromEnvironment(new TestLogger());
+                settings.Should().NotBeNull("Test setup error: TFS environment variables have not been set correctly");
+                settings.BuildEnvironment.Should().Be(BuildEnvironment.NotTeamBuild, "Test setup error: build environment was not set correctly");
+
+                var preProcessor = new TeamBuildPreProcessor(mockFactory, logger);
+
+                // Act
+                var success = preProcessor.Execute(CreateValidArgs("key", "name", "1.0", "organization")).Result;
+                success.Should().BeTrue("Expecting the pre-processing to complete successfully");
+            }
+
+            mockTargetsInstaller.Verify(x => x.InstallLoaderTargets(workingDir), Times.Once());
+
+            mockServer.AssertWarningWritten("version is below supported");
+        }
+
+        [DataTestMethod]
+        [DataRow("7.9.0.5545", DisplayName = "7.9 LTS")]
+        [DataRow("8.0.0.18670", DisplayName = "SonarCloud")]
+        [DataRow("8.8.0.1121")]
+        [DataRow("9.0.0.1121")]
+        [DataRow("10.15.0.1121")]
+        public void PreProc_EndToEnd_ShoulNotdWarn_SonarQubeDeprecatedVersion(string sqVersion)
```

Comment:
Copy/paste artifact (additional `d` removed):
```suggestion
        public void PreProc_EndToEnd_ShoulNotWarn_SonarQubeDeprecatedVersion(string sqVersion)
```